### PR TITLE
Bump utility dependencies

### PR DESCRIPTION
**Pull request by thekp**
Resolves n/a

**Overall change:** more bumping
we need to be using the latest version of dependencies in each package to avoid duplication 

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-story-promo/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 2.3.5   | [PR#1685](https://github.com/bbc/psammead/pull/1685) Bump dependencies |
```
Comment (dr3) [inline packages/components/psammead-story-promo/CHANGELOG.md:6]:
```suggestion
| 2.3.3   | [PR#1685](https://github.com/bbc/psammead/pull/1685) Bump dependencies |
```

File: packages/components/psammead-consent-banner/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 2.2.1   | [PR#1685](https://github.com/bbc/psammead/pull/1685) Bump dependencies |
```
Comment (dr3) [inline packages/components/psammead-consent-banner/CHANGELOG.md:6]:
```suggestion
| 2.2.2   | [PR#1685](https://github.com/bbc/psammead/pull/1685) Bump dependencies |
```

**Review by dr3 — CHANGES_REQUESTED** [review]

**Review by dr3 — APPROVED** [review]

**Review by jamesbhobbs — APPROVED** [review]

File: packages/components/psammead-story-promo/CHANGELOG.md (hunk 1 above)
Comment (DenisHdz) [reply packages/components/psammead-story-promo/CHANGELOG.md:6]: Where has the `2.3.3` gone? 🙈

**Pull request merged**